lib/rules/web/admin_console/4.6/overview.xyaml

```diff
@@ -216,4 +216,63 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
-
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
+send_key_to_workloads:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: send_keys <key>
+check_workload_selected:
+  elements:
+  - selector:
+      xpath: //div[@class='list-group-item project-overview__item project-overview__item--<workload> project-overview__item--selected']
+click_workload_details:
+  element:
+    selector:
+      xpath: //button[contains(text(),'Details')]
+    op: click
+check_workload_name:
+  elements:
+  - selector:
+      xpath: //div[@class='resource-overview__summary']//button[text()='Name']
+  - selector:
+      xpath: //div[@class='resource-overview__summary']//dd[text()='<workload_name>']
+check_hint_missing_on_filter:
+  elements:
+  - selector:
+      xpath: //span[@class='form-control-feedback form-control-feedback--keyboard-hint']//kbd
+    missing: true
+check_workload_sidebar_missing:
+  elements:
+  - selector:
+      xpath: //div[@class='overview__sidebar overview__sidebar-enter-done']
+    missing: true
+check_shortcuts_on_workloads:
+  params:
+    workload: <workload>
+    workload_name: <workload_name>
+    keys: <key>
+  action: click_workloads_tab
```

Comment:
`goto_project_resources_page` already redirected user to the Workloads page, so we don't need `click_workloads_tab` again

Reply:
@yapei The context focus is on filter name input box and hence the hot keys are not working. Hence I had to click the workloads link one more time to take the context focus away from the filter name box.